### DISCUSSION TITLE
Welches Update läuft stabil und kann ich nehmen?

### DISCUSSION THREAD

**Discussion by Dave24282** [tbnobody/OpenDTU#956] [Q&A]
![Screenshot_20230524_195307_edit_275495233449628](https://github.com/tbnobody/OpenDTU/assets/134503480/3fe4bcfc-da3b-4237-8645-72fb993cbf7e)
Hallo habe diese Version und sie läuft stabil aber er sagt ich soll updaten nun meine frage welche version läuft stabil und welche soll ich nehmen?

**Comment by Solarteur** [comment]
Bei mir läuft die v23.5.23 stabil - genau wie die vorhergehenden Versionen auch. 
Wenn die bestehende Firmware vor dem 15.03.2023 aufgespielt wurde - wovon ich ausgehe - bitte diese Hinweise beachten:

!! IMPORTANT UPGRADE NOTES !!

If you are upgrading from a version before 15.03.2023 you have to upgrade the partition table of the ESP32. Please follow the [this](https://github.com/tbnobody/OpenDTU/blob/master/docs/UpgradePartition.md) documentation!

**Reply by Dave24282** [reply]
Also müsste ich alle 4 Dateien nehmen und nicht über dtu Firmware Aktualisierung nur die Datei aktualisieren opendtu-generic.bin das geht nicht?

Wie kann ich eigentlich den aktuellen Stromverbrauch sehen?

**Reply by Solarteur** [reply]
Ja, einmalig alle vier Dateien wie beschrieben verwenden. Danach geht wieder OTA.

Wenn mit dem aktuellen Stromverbrauch der Stromverbrauch der DTU gemeint ist: Externe Messung. Bei mir ergibt die Messung auf der 230V-Seite ca. 0,9W.

**Reply by Dave24282** [reply]
Ich meine den Hausverbrauch.

**Reply by Solarteur** [reply]
OpenDTU zeigt Leistung und Ertrag des/der angeschlossenen Wechselrichter(s) an (-->Einspeisung). Zur Anzeige des **Verbrauchs** des gesamten Hauses sind weitere Geräte notwendig (z. B. Shelly EM3, Geräte, welche die Daten des Stromzählers auswerten).

**Reply by Dave24282** [reply]
Ich habe mir jetzt den Lesekopf für den Zähler geholt mit allen... Kennst du dich auch mit tasmota aus?

**Reply by Solarteur** [reply]
Nein, überhaupt nicht. Für den Gesamt-Stromverbrauch habe ich einen Shelly EM3 im Einsatz und nutze den weiterhin.

**Reply by Dave24282** [reply]
Okay danke für die schnelle Antwort

Kannst du mir vielleicht gleich sagen Was die neuen Features gegenüber meiner Version die 4.4.3

**Comment by DPO99** [comment]
Never Change a running system ;). Warum updaten wenns läuft und Du keines der neuen Features brauchst?

**Reply by Dave24282** [reply]
Was wären die neuen Features gegenüber meiner Version die 4.4.3.
